bin/plotting/pycbc_plot_qscan

```diff
@@ -92,94 +92,108 @@ opts = parser.parse_args()
 
 logging.basicConfig(format='%(asctime)s %(message)s', level=logging.INFO)
 
-#opts.low_frequency_cutoff = opts.f_low
-strain = pycbc.strain.from_cli(opts, pycbc.DYN_RANGE_FAC)
 
 if opts.center_time is None:
     center_time = (opts.gps_start_time + opts.gps_end_time) / 2.
 else:
     center_time = opts.center_time
 
-if opts.qtransform_frange_upper is None and \
-        opts.qtransform_frange_lower is None:
-    curr_frange = (30, opts.sample_rate / 4.)
-elif opts.qtransform_frange_upper is None or \
-        opts.qtransform_frange_lower is None:
-    err_msg = 'Must provide either both --qtransfor-frange-upper and '
-    err_msg += '--qtransfor-frange-lower or neither option.'
-    raise ValueError(err_msg)
+if center_time == -1.0:
+    fig, axes = plt.subplots(1,1)
+    fig.add_subplot(111, frameon=False)
+    plt.tick_params(labelcolor='none', top='off', bottom='off', left='off',
+                    right='off')
+    plt.grid(False)
+    plt.xlabel('Time from {:.3f} (s)'.format(opts.center_time))
+    plt.ylabel('Frequency (Hz)')
+    title = 'No triggers or this IFO'
+    pycbc.results.save_fig_with_metadata\
+    (fig, opts.output_file, cmd=' '.join(sys.argv), fig_kwds={'dpi': 150},
+     title=title, caption=opts.plot_caption)
```

Comment:
Same comment about indenting and `sys.exit(0)` here to avoid having to make a large change and indent a lot of code.

Reply:
Done